packages/manager/apps/pci-load-balancer/src/pages/create/store.ts

```diff
@@ -21,19 +14,24 @@ type TStep = {
 };
 
 export enum StepsEnum {
-  SIZE = 'SIZE',
   REGION = 'REGION',
+  SIZE = 'SIZE',
   IP = 'IP',
   NETWORK = 'NETWORK',
   INSTANCE = 'INSTANCE',
   NAME = 'NAME',
 }
 
+export enum FloatingIpSelectionId {
```

Comment:
If this enum is shared over the whole project, it should be declared at a contant level, to prevent from importing the store each time